src/components/index/IndexBanner.jsx

```diff
@@ -45,7 +45,7 @@ export default function IndexBanner({data}) {
             number="1.2 K"
           />
         </div>
-        <Space size={32} className={styles.buttons}>
+        <Space size={32} className={styles.buttons} onClick={() => { throw new Error('sentry test');}}>
```

Comment:
Do you want to temporarily include this testing code in PROD?

Reply:
If you do, don't forget to remove it after your testing.

Reply:
That's a mistake, removing it.